conda/gateways/repodata/jlap/fetch.py

```diff
@@ -358,9 +358,9 @@ def request_url_jlap_state(
             state.set_has_format("jlap", True)
             need_jlap = False
         except ValueError:
-            log.info("Checksum not OK")
-        except IndexError as e:
-            log.info("Incomplete file?", exc_info=e)
+            log.info("Checksum not OK on JLAP range request. Retry with complete JLAP.")
+        except IndexError:
+            log.exception("Incomplete file?")
```

Comment:
This error really is unexpected. It could happen if the remote file is an invalid jlap with fewer than (leading checksum, footer json, trailing checksum) lines.

Reply:
Seeing your fix now 👍🏻